schedule/yast/yast2_ncurses/yast2_ncurses_gnome.yaml

```diff
@@ -29,12 +29,15 @@ conditional_schedule:
         - console/yast2_lan_hostname
         - console/yast2_settings
         - console/yast2_dns_server
+        - console/yast2_nfs_server
         - console/yast2_nfs_client
         - console/yast2_snapper_ncurses
       aarch64:
         - console/yast2_lan
+        - console/yast2_nfs_server
       ppc64le:
         - console/yast2_lan
+        - console/yast2_nfs_server
```

Comment:
we should include it in s390x despite it was not schedule it in functional because it would probably work just fine, its companion yast2_nfs_client is there.

Reply:
i will try. but because i had some failures before on s390x i will file another ticket to address the problem. is that ok?

Reply:
please, point to the failures, we can handle here. Test coverage should be uniform unless we encounter something really weird that is better to handle aside.

Reply:
I would rather not. keep it simple. this is about the moving to the yast group the scenario. failure can be seen here https://openqa.suse.de/tests/4676017

Reply:
fine, seems that already contains directory and the test fails in needles.